### ISSUE TITLE
Replaced by wyoming-satellite?

### ISSUE THREAD

**Issue by raetha** [synesthesiam/homeassistant-satellite#83]
Had this been replaced by https://github.com/rhasspy/wyoming-satellite?

I just stumbled across that project which appears to have been updated more recently, and is part of the overall Wyoming project. It seems very similar, though includes auto discovery by Home Assistant.

I believe the author is the same and am trying to figure out which one makes the most sense to be building off of.

Thanks!

**Comment by donburch888** [comment]
Not according to my understanding ... though I admit it can be confusing ;-)  

Homeassistant-satellite lets you use a linux computer (such as a Raspberry Pi) as a satellite to HA Voice Assist running on another computer with Home Assistant installed.  Normally in this configuration you will use Wyoming which is installed as a Home Assistant add-on to do the coordinating between the various components, and so wyoming-satellite is NOT needed. 
If you want to do the wakeword detection on your satellite RasPi you can also run [openwakeword](https://github.com/rhasspy/wyoming-openwakeword) as per [my post](https://community.home-assistant.io/t/raspberry-pi-as-a-ha-voice-assist-satellite/632671)

My understanding is that wyoming-satellite is for anyone who wants to run wyoming, but NOT under Home Assistant.  That would be for Rhasspy v3, or if they want to run each component on separate physical or virtual machines.  Mike really is focussed on providing a kit of tools people can combine in various ways to fit whatever project they are working on.

**Comment by raetha** [comment]
Got it, I think! I just noticed in the recent Home Assistant patch notes a mention of support for Wyoming Satellites on RPis, and linking to the other repo. HA even has auto discovery for them. So it looked like an upgraded version of this, which also has a few more features like callback scripts, etc. I’m good with either path, just want to make sure I’m using what will be updated and has the most community support! :)

In my case I run HA in Docker, and am running separate Docker containers for each Wyoming service. So it sounds like this might better fit my situation anyway. Ultimately looking to create an RPi satellite that does local wake word, and acts as a media player using Mopidy, MPD, or Snapcast.

**Comment by chatziko** [comment]
My understanding is that eventually it will be replaced, as wyoming offers a better communication protocol with the satellite. The main advantage for the moment is that wyoming-satellite will appear as a device in HA.

See https://github.com/home-assistant/core/pull/104759

Of course HA uses open protocols, so eventually more satellites with different features will be developed, choice is a good thing.

**Comment by synesthesiam** [comment]
Yes, wyoming-satellite will replace this project eventually. It's not fully stable yet, so expect some issues.

Besides discovery, wyoming-satellite uses Wyoming services for audio input/output (and wake word detection). I want to move the great work on pulseaudio, etc. people have done here to their own services so it's easier to maintain.

**Comment by Solarunit** [comment]
>In my case I run HA in Docker, and am running separate Docker containers for each Wyoming service. So it sounds like this might better fit my situation anyway.

I run HA in Docker too, so the only way to get usb-mic working was to add another container `homeassistant-satellite` on the same PC.
 
Now I wonder cam I use `wyoming-satellite` instead of `homeassistant-satellite` for my case?

**Comment by andreasbrett** [comment]
> Yes, wyoming-satellite will replace this project eventually. It's not fully stable yet, so expect some issues.
> 
> Besides discovery, wyoming-satellite uses Wyoming services for audio input/output (and wake word detection). I want to move the great work on pulseaudio, etc. people have done here to their own services so it's easier to maintain.

Could you open up the docker images on the Docker hub? I'd love to test it already (yes, I'm kind of the kid that searches the house for all the hidden christmas presents...). Unfortunatley the images are not public yet.

```
Error response from daemon: pull access denied for rhasspy/wyoming-snd-external, repository does not exist or may require 'docker login': denied: requested access to the resource is denied```
